code/modules/mob/living/carbon/human/species/zombie.dm

```diff
@@ -24,6 +24,7 @@
 	stun_reduction = 10
 	knock_out_reduction = 5
 	has_organ = list()
+	set_languages(list(LANGUAGE_ZOMBIE))
```

Comment:
set this on init

Reply:
> set this on init

This is how all the other species do it I think

Reply:
![image](https://github.com/cmss13-devs/cmss13/assets/41448081/bde9a739-5b29-476e-b8ca-57e033748d90)

it's not

Reply:
Implemented; my bad I was tired when I made this